lib/THC/THCCachingAllocator.cpp

```diff
@@ -158,12 +158,12 @@ struct THCCachingAllocator
     allocated_blocks.erase(it);
 
```

Comment:
And , yes, the largest block size parameter is now unused.
@soumit, @colesbury: we can either purge it or hook it up for the user to actually get this info, let me know which way you prefer.